internal/envoy/listener.go

```diff
@@ -28,6 +28,18 @@ import (
 	"github.com/heptio/contour/internal/dag"
 )
 
+// HTTPDefaultIdleTimeout sets the idle timeout for HTTP connections
+// here and in the e2e tests.
+const HTTPDefaultIdleTimeout = 60
```

Comment:
Rather than HTTPDefaultIdleTimeout * time.Second lower down, why not multiply this constant by time.Second on this line then it is clear this is 60seconds, not 60 something’s which have the be scaled at the place they are used.

Reply:
That makes sense. Will do.